sqlpage/templates/tracking.handlebars

```diff
@@ -0,0 +1,25 @@
+<div class="card my-2 col-md-{{default width 12}}">
+	<div class="card-body">
+		<div class="d-flex align-items-center">
+			<div class="subheader">{{title}}</div>
+		</div>
+		<div class="d-flex align-items-baseline remove-bottom-margin">
+			<div class="h1 mb-3 me-2">{{#if information}}{{information}}{{/if}}</div>
+		</div>
+		<div class="d-flex align-items-baseline">
+			<div class="mb-3 me-2
+				{{#if description_md}} remove-bottom-margin">{{{markdown description_md}}}
```

Comment:
I think you can simply apply  remove-bottom-margin regardless. If the div has no children (not in markdown mode), it will simply have no effect.

Reply:
yes. I've simplified the template